tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+expense of depth). Subsequent tutorials address each of these topics in greater
+detail. We begin by importing the necessary Python modules:
+"""
 
-mne.set_log_level('INFO')
+import os
+import numpy as np
+import mne
 
-##############################################################################
-# You can set the default level in every session by setting the environment
-# variable "MNE_LOGGING_LEVEL", or by having mne-python write preferences to a
-# file with::
+###############################################################################
+# Loading data
+# ^^^^^^^^^^^^
 #
-#     >>> mne.set_config('MNE_LOGGING_LEVEL', 'WARNING')
+# MNE-Python data structures are based around the FIF file format from
+# Neuromag, but there are reader functions for :ref:`a wide variety of other
+# data formats <data-formats>`. MNE-Python also has interfaces to a
+# variety of :doc:`publicly available datasets <../../manual/datasets_index>`,
+# which MNE-Python can download and manage for you.
 #
-# Note that the location of the mne-python preferences file (for easier manual
-# editing) can be found using:
-
-print(mne.get_config_path())
-
-##############################################################################
-# By default logging messages print to the console, but look at
-# :func:`mne.set_log_file` to save output to a file.
+# We'll start this tutorial by loading one of the example datasets (called
+# ":ref:`sample-dataset`"), which contains EEG and MEG data from one subject
+# performing an audiovisual experiment, along with structural MRI scans for
+# that subject. The :func:`mne.datasets.sample.data_path` function will
+# automatically download the dataset if it isn't found in one of the expected
+# locations, then return the directory path to the dataset (see the
+# documentation of :func:`~mne.datasets.sample.data_path` for a list of places
+# it checks before downloading).
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, preload=False)
+
+###############################################################################
+# .. note::
 #
-# Access raw data
-# ^^^^^^^^^^^^^^^
-
-from mne.datasets import sample  # noqa
-data_path = sample.data_path()
-raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
-print(raw_fname)
-
-##############################################################################
-# .. note:: The MNE sample dataset should be downloaded automatically but be
-#           patient (approx. 2GB)
+#     :func:`~mne.io.read_raw_fif` takes a ``preload`` parameter, which
+#     determines whether the data will be copied into RAM or not. Some
+#     operations (such as filtering) require that the data be preloaded, but it
+#     is possible use ``preload=False`` and then copy raw data into memory
```

Comment:
not sure if preload logic fits into a basic tutorial. I would say it's only for advanced people who want to save memory. Most beginners wouldn't care. I would even go so far as to hide preload=False on L42 above

Reply:
kind of important feature, though, so perhaps worth pointing out? how about:

... determines whether to load the data from disk into memory. Some operations, [...] , but with ``preload=False`` you can wait to do so until it is necessary. The :meth:`~mne.io.Raw.load_data` method can then be used to  load the data.